students/philip_behrend/lesson02/charges_calc.py

```diff
@@ -0,0 +1,86 @@
+'''
+Returns total price paid for individual rentals 
+'''
+import argparse
+import json
+import datetime
+import math
+import logging
+import traceback
+
+def set_logging(log_level):
+    log_format = "%(asctime)s %(filename)s:%(lineno)-4d %(levelname)s %(message)s"
+    log_file = datetime.datetime.now().strftime("%Y-%m-%d")+'.log'
+    options = {
+        0: logging.CRITICAL,
+        1: logging.ERROR,
+        2: logging.WARNING,
+        3: logging.DEBUG
+    }
+
+    try: 
+        log_level = options.get(int(log_level))
```

Comment:
This is a good idea, except that using the _get()_ method will return _None_ for invalid keys instead of raising a _KeyError_ exception. Instead, use square brackets:

`log_level = options[int(log_level)]`

There a minor coding error that will show up once you fix the one above. Should be an easy fix.